test/functionsMetaData.spec.ts

```diff
@@ -103,3 +105,176 @@ describe('writeFunctionConfiguration', () => {
     expect(actual).toEqual(expected)
   })
 })
+
+describe('writeEdgeFunctionConfiguration', () => {
+  afterEach(() => {
+    mock.restore()
+  })
+
+  it('should write the configuration for an edge function using node modules version of @netlify/plugin-nextjs', async () => {
```

Comment:
These tests are testing more than `writeEdgeFunctionConfiguration` (as the function doesn't write anything). How about splitting up the tests?

Reply:
Hey, so I ended up removing the tests you referenced based on our convo today. Since we are now using the `getPluginVersion` within the `writeFunctionsConfiguration` the original tests that Nick worked on should be enough to check and make sure it is working as expected.

Reply:
Thanks! I won't stop this PR from merging but I think it would be good to get some tests around the new generator code that's been added (which uses the plugin version now). This might be more work to get them set up as none exist right now for the edge functions so maybe raise an issue to add them?

Reply:
Since we're adding this code to generate meta data for edge functions, I think it'd be good to add the tests in this PR. We have our 1:1 today @taty2010, so we can take a peek at this then.

Reply:
That would be great. Thanks for helping @nickytonline.